tartufo/util.py

```diff
@@ -44,29 +45,49 @@ def convert_regexes_to_rules(regexes: Dict[str, Pattern]) -> Dict[str, Rule]:
     }
 
 
-def echo_issues(
+def echo_result(
     issues: "List[Issue]",
-    as_json: bool,
+    options: "types.GlobalOptions",
+    scanner: "ScannerBase",
```

Comment:
This makes me wonder. If we're passing in the scanner, why are we will passing in the issues? I believe we should be able to access `scanner.issues` for the same list. IIRC I set it up so that `.issues` is a cached property, which gets populated by `.scan()`. Would that make more sense here?

Also, why pass in the `now` param rather than just generating it inside this function? It seems like an almost unnecessary extra parameter.

Reply:
Removed the `now` parameter and instantiating this timestamp in the function instead. My thought here was to have equality between a directory name for `output_dir` and the timestamp in the output. However, both of these happen after the actual scanning of the code, so should be very similar in time. 
I can't see that issues are accessible by .issues, but from `._issues` which seems less like it's intended to be exposed to external members. Letting both `issues` and `scanner` be separate arguments for now, but open to arguments 🧐

Reply:
https://github.com/godaddy/tartufo/blob/6c29000f63a2b10c457d2e7023a66561c833f2cf/tartufo/scanner.py#L119-L130 -- this can be used to access a cached copy of the issues, which is set here at the end of the scan: https://github.com/godaddy/tartufo/blob/6c29000f63a2b10c457d2e7023a66561c833f2cf/tartufo/scanner.py#L264-L265